app/mailers/order_delivery_mailer.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class OrderDeliveryMailer < ApplicationMailer
+  def delivery_email
```

Comment:
I find this name a bit misleading, what would you say about `delivery_confirmation_email` instead?

Reply:
ok, fixed